<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="SearchButtom">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>40</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QListWidget" name="Result">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>91</y>
      <width>751</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="LastPage">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>540</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>LastPage</string>
    </property>
   </widget>
   <widget class="QPushButton" name="NextPage">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>540</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>NextPage</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Safe">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="SearchBar">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
